frontend/components/panels/SurplusAuctionRestartPanels.stories.js

```diff
@@ -0,0 +1,88 @@
+import { storiesOf } from '@storybook/vue';
+import faker from 'faker';
+import SurplusAuctionRestartPanel from '~/components/panels/SurplusAuctionRestartPanel';

```

Comment:
You currently get an error for this line:
```
Could not locate module ~/components/panels/SurplusAuctionRestartPanel
```

That's because your files are actually called `SurplusAuctionRestartPanels`, so plural. Please rename both the component and the story file to just `SurplusAuctionRestartPanel` 🙂